shared/js/background/redirect.es6.js

```diff
@@ -48,7 +48,7 @@ function handleRequest (requestData) {
 
     let thisTab = tabManager.get(requestData)
 
-    const blockingEnabled = utils.isFeatureEnabled('contentBlocking') && !utils.isFeatureBrokenForURL(thisTab.url, 'contentBlocking')
+    let blockingEnabled = utils.isFeatureEnabled('contentBlocking')
```

Comment:
We don't use this variable until the `else` that starts on line 77. We can just move the whole line down there after the `thisTab` check that is done at the start of that block.